packages/data-context/src/actions/EventCollectorActions.ts

```diff
@@ -25,10 +25,10 @@ export class EventCollectorActions {
 
   async recordEvent (event: CollectableEvent): Promise<boolean> {
     try {
-      const dashboardUrl = this.ctx.cloud.getDashboardUrl(cloudEnv)
+      const cloudUrl = this.ctx.cloud.getCloudUrl(cloudEnv)
```

Comment:
Nice, glad we are changing the vars, not just the strings, it'll be much clearer.